Documentation/General/finalReport/appendix-neuralnetwork.tex

```diff
@@ -0,0 +1,42 @@
+\chapter{Neural Networks}
+\label{appendix-neural-networks}
+
+One of the bigger issues we had to deal with in MoodCat was to classify the mood of a song.
+The initial plan was to create a neural network that could do this.
+
+Some of the MoodCat team members have followed a course Computational Intelligence,
+where they learned different learning algorithms to solve different kinds of problems.
+One widely used technique to classify objects by using the objects' features as inputs for the neural network.
+
+\section{Structure of the neural network}
+Neural networks consist of an input layer, an output layer and a number of hidden layers.
+Each layer consists of a number of nodes (See also figure \ref{neural-network-graph}).
+The nodes in the input layer are fed with \textit{features}.
+For the classification of songs these would be all sorts of information of the song that can be represented as some form of numeric metrics, for example: the amount of beats per minute (BPM), the average loudness or tonality of the song.
+The key of a song is not an obvious metric as it cannot be represented on a linear scale: music in A has not more in common with B than C or even G, while on a linear scale A would deviantly have more in common with B than G.
+The network could however map the input for each key: if a song is written in A, the A input is 1 and the other key inputs are 0.
+
+The nodes in the hidden layers have nodes with weights for each node in the next layer,
+and the output layers would have a node for each mood, the output nodes output a number representing how likely the song
```

Comment:
The is still some debate on this and we should probably point that out. @Azkle @GijsWeterings @JeremybellEU